.gitlab-ci.yml

```diff
@@ -29,6 +29,9 @@ build:
       - R_ENVIRONMENT: ["rstudio", "rstudio-local"]
         R_VERSION: ["4.1.0", "4.1.1"]
```

Comment:
You just need to update this list with `4.1.2`, @tomszosz

Reply:
gitlab file is removed